packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.tsx

```diff
@@ -50,7 +51,9 @@ export class DocumentCard extends BaseComponent<IDocumentCardProps, any> {
         onClick={actionable ? this._onClick : undefined}
         style={style}
       >
-        {children}
+        <Link as="a" {...linkProps}>
```

Comment:
We don't need a link here, I was referring to the DocumentCardPreivew

![image](https://user-images.githubusercontent.com/1434956/45571262-8bdd5980-b81a-11e8-8e70-2e97b4753c00.png)

![image](https://user-images.githubusercontent.com/1434956/45571278-97308500-b81a-11e8-8c2b-01f8a198de4c.png)


That a tag should be a Link, and IDocumentCardPreviewImage could then have linkProps to pass into it.